app/src/main/java/ch/hikemate/app/ui/map/RunHikeScreen.kt

```diff
@@ -263,30 +256,41 @@ fun RunHikeBottomSheet(
 
             DetailRow(
```

Comment:
Suggestion: I noticed only now that you use the `DetailRow` component of the `HikeDetailsScreen`, should it become a component of its own in the `components` package (creating a new file `DetailRow.kt` with the composable inside, moving the test tags to that file, and so on)?

Reply:
fair point, I've moved it and updated the test tags